Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -106,6 +110,8 @@ private double getTagTrendingScore(Collection<String> tags) {
         int numTags = tags.size();
         for (String tag: tags) {
             double tagScore = HardCodedCharitiesAndTags.tagScores.get(tag);
+            //double tagScore = tag.getTrendingScore();
+            //store tagScore in db and retreive score for specific tag when required
```

Comment:
is it again to hardcoding? Add a TODO() comment.

Reply:
added comment.